openapi/p4pa-pu-bff.openapi.yaml

```
@@ -0,0 +1,106 @@
+openapi: 3.0.1
+info:
+  title: API Documentation
+  description: OpenAPI documentation
+  version: 1.0.0
+servers:
+  - url: "http://localhost:8080"
+    description: Generated server url
+paths:
+  /organizations:
+    get:
+      tags:
+        - Organizations
+      summary: "Retrieve the list of organizations"
+      operationId: getOrganizations
+      security:
+        - bearerAuth: [ ]
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/OrganizationDTO'
+        '401':
+          description: Unauthorized - Missing or invalid token
+        '403':
+          description: Forbidden - Access denied
+        '500':
+          description: Internal Server Error
+  /info/brokerConfig:
```

let's use the naming conventions based on entities to retrieve, as for example, now we are accessing to `brokers` entity (we use plural in order to handle also findAll useCases).
This API is not returning the entire model, but it's specialized model/representation/nestedInfo `config`
```suggestion
  /brokers/config:
```